classes/submission/Submission.php

```diff
@@ -44,6 +48,23 @@ class Submission extends PKPSubmission
     // Get/set methods
     //
 
+    /**
+     * @copydoc \PKP\submission\PKPSubmission::getStatusMap()
+     */
+    public function &getStatusMap()
+	{
+		static $statusMap = null;
```

Comment:
Just syntax, but I think you can use:
```php
static $statusMap = parent::getStatusMap() + [
    ...
];
return $statusMap;
```

Reply:
as we moving the new ones to publication , it's not needed anymore .